tests/integration-tests/tests/dcv/test_dcv/test_dcv_configuration/pcluster.config.ini

```diff
@@ -11,8 +11,14 @@ vpc_settings = parallelcluster-vpc
 scheduler = {{ scheduler }}
 master_instance_type = {{ instance }}
 compute_instance_type = {{ instance }}
+{% if scheduler == "awsbatch" %}
+min_vcpus = 32
```

Comment:
Just curious: do you know why are we spinning up the compute fleet in DCV tests?

Reply:
Good question, we can get rid of them